com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -72,6 +72,12 @@ public class NetworkConfig
         [Tooltip("The amount of times per second the receive queue is emptied from pending incoming messages")]
         public int ReceiveTickrate = 64;
 
+        /// <summary>
+        /// Duration in seconds between network ticks.
+        /// </summary>
+        [Tooltip("Duration in seconds between network ticks")]
+        public float NetworkTickIntervalSec = 0.050f;
```

Comment:
I think we should also update `NetworkTickSystem.k_DefaultTickIntervalSec` according to this default as well.

Reply:
(super minor) also, we might call this as plain `TickIntervalSec` since "Network" is pretty obvious in this context.

Reply:
this?
```
-        private const float k_DefaultTickIntervalSec = 1 / 60f; // Defaults to 60 ticks second
+        private const float k_DefaultTickIntervalSec = 0.05f; // Defaults to 20 ticks second
```

Reply:
yup!